Assignment.java

```diff
@@ -0,0 +1,71 @@
+package package1;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class Assignment {
+	
+	WebDriver driver;
+
+	@Before
+	public void setupMethod() {
+		 System.setProperty("webdriver.chrome.driver","C:\\jar files\\chromedriver\\chromedriver.exe");
+		 driver = new ChromeDriver();
+		 driver.manage().window().maximize();
+	}
+		
+	@Test 
+	public void scenario2() {
```

Comment:
you are missing scenario 1 and try to follow the following naming convention in this page https://dzone.com/articles/7-popular-unit-test-naming, follow pattern 2. Try doing this for all test cases